boards/DCM/Src/App/states/App_FaultState.c

```diff
@@ -37,8 +37,7 @@ static void FaultStateRunOnTick100Hz(struct StateMachine *const state_machine)
 
     App_CanTx_SetPeriodicSignal_TORQUE_REQUEST(can_tx_interface, 0.0f);
 
-    if (!App_SharedStates_HasInverterFaulted(can_rx_interface) &&
-        !App_SharedErrorTable_HasAnyCriticalErrorSet(error_table))
+    if (!App_HasInverterFault(can_rx_interface) && !App_SharedErrorTable_HasAnyCriticalErrorSet(error_table))
```

Comment:
Could even shorten it further to 

```
if (App_AllStatesRunOnTick100Hz(state_machine))
{
    App_SharedStateMachine_SetNextState(state_machine, App_GetInitState());
}
```

Reply:
@timbitss This won't run the fault state code then since App_AllStatesRunOnTick100Hz will return false

Reply:
AH wait nvm i see what you mean now, yeah nice